aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -1088,7 +1088,13 @@ async def get_connection_targets(
 
                     targets = await self.fetch_connection_targets(connection)
 
-                    await entry.set_result([row.serialize() for row in targets], 3600)
+                    if connection.state == ConnRecord.State.COMPLETED.rfc160:
```

Comment:
Just to check, because I don't remember how the state is managed here, but would this comparison be expected to work for connections and did-exchange protocols?

Reply:
Yep, that's right -- the `state` attribute of connection records will always reflect the RFC 160 state whereas `rfc23_state` reflects the DID Exchange state, regardless of which protocol is actually used.